.npmrc

```diff
@@ -1,3 +1,9 @@
 shamefully-hoist=false
 strict-peer-dependencies=false
 node-linker=isolated
+
+# GitHub Packages configuration for @cowprotocol scope
+@cowprotocol:registry=https://npm.pkg.github.com
```

Comment:
How does that behave when trying to publish "real" versions to NPM?

Reply:
It's possible that breaks, let me try to see if I can add the registry just in the action

Reply:
Removed that 610a6cc3ba320e3b88f34cfd8ad28e810cc20ae8